sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -420,26 +420,35 @@ async def negotiate(self):
         if not self.sslopts:
             return
         await self.write(TLS_HEADER_FRAME)
-        channel, returned_header = await self.receive_frame(verify_frame_type=None)
+        _, returned_header = await self.receive_frame(verify_frame_type=None)
         if returned_header[1] == TLS_HEADER_FRAME:
             raise ValueError(
-                "Mismatching TLS header protocol. Excpected: {}, received: {}".format(
-                    TLS_HEADER_FRAME, returned_header[1]
-                )
+                f"""Mismatching TLS header protocol. Expected: {TLS_HEADER_FRAME!r},"""
+                """received: {returned_header[1]!r}"""
             )
 
 
 class WebSocketTransportAsync(AsyncTransportMixin):
-    def __init__(self, host, port=WEBSOCKET_PORT, connect_timeout=None, ssl=None, **kwargs):
+    def __init__(
+        self,
+        host,
+        *,
+        port=WEBSOCKET_PORT,  # pylint: disable=unused-argument
```

Comment:
@kashifkhan - looks like port isn't being used at all. checking that it's intentional?

Reply:
It was never being used from the get go, I think we should append to the end of the host 
`self.host = f"{self.host}:{port}"`